sources/packages/web/src/views/aest/institution/ApplicationOfferingChangeRequestForm.vue

```diff
@@ -147,6 +177,33 @@ export default defineComponent({
         ),
       };
     });
+
+    const assessApplicationOfferingChangeRequest = async (
+      applicationOfferingChangeRequestStatus: ApplicationOfferingChangeRequestStatus,
+    ) => {
+      const responseData =
+        await assessApplicationOfferingChangeRequestModal.value.showModal(
+          applicationOfferingChangeRequestStatus,
+        );
+      if (responseData) {
+        try {
+          await ApplicationOfferingChangeRequestService.shared.updateApplicationOfferingChangeRequestStatus(
+            props.applicationOfferingChangeRequestId,
+            responseData as ApplicationOfferingChangeAssessmentAPIInDTO,
+          );
+          router.push({
+            name: AESTRoutesConst.ASSESSMENTS_SUMMARY,
+          });
+          const snackbarMessage =
```

Comment:
Is the intention of having the `snackbarMessage` declared as a const to improve the code readability?
I am not sure is it is worth it in this scenario.